db/controllers/comments.controller.js

```diff
@@ -0,0 +1,14 @@
+const { insertComment } = require('../models/comments.model')
+
+function postCommentWithId(req, res, next) {
+    const insert = req.body
+    const id = req.params.article_id
+
+    insertComment(insert, id).then(({rows}) => {
+
+        res.status(201).send(rows[0])
```

Comment:
Please return the data on an appropriate key - all the data manipulation and access should be done in the model